src/app/page.js

```diff
@@ -1,16 +1,14 @@
 "use client";
 
+import { useEffect, useState } from "react";
 import { Fragment, useState } from "react";
 
 import SideNav from "@/components/SideNav";
 import Repos from "@/components/Repos";
 import Activity from "@/components/Activity";
 import Stats from "@/components/Stats";
 import Toast from "@/components/Toast";
-import { useSearchParams } from "next/navigation";
-import { Menu, Transition } from "@headlessui/react";
-import { ChevronUpDownIcon } from "@heroicons/react/24/outline";
-import { classNames } from "@/utils/classNames";
```

Comment:
Not sure why these are removed? The page will break, please test

Reply:
It was removed by me while fixing the conflict.